project/AppDependencies.scala

```diff
@@ -26,6 +30,6 @@ object AppDependencies {
     "org.pegdown" % "pegdown" % "1.6.0" % "test",
     "org.jsoup" % "jsoup" % "1.12.1" % "test",
     "com.typesafe.play" %% "play-test" % PlayVersion.current % "test",
-    "org.mockito" % "mockito-core" % "3.0.0" % "test"
+    "org.mockito" % "mockito-core" % "3.0.0" % "test",
```

Comment:
Unnecessary comma at the end of the line

Reply:
My mistake!!!! :)
Fixed.